pkg/storage/stores/shipper/compactor/deletion/delete_requests_manager_test.go

```diff
@@ -54,6 +54,14 @@ func (m mockDeleteRequestsStore) Stop() {
 	panic("implement me")
 }
 
+func (m mockDeleteRequestsStore) GetCacheGenerationNumber(ctx context.Context, userID string) (string, error) {
+	panic("implement me")
+}
```

Comment:
should this be implemented as part of this PR?

Reply:
Nope. This is just to satisfy the interface for another component. I made it clearer by having the mock embed the interface and only override the used method

Reply:
we never need to call this function as a result of a delete test? :thinking: